devito/arch/archinfo.py

```diff
@@ -377,6 +377,17 @@ def get_cuda_path():
     return None
 
 
+@memoized_func
+def get_hip_path():
+    # *** First try: via commonly used environment variables
```

Comment:
But there is no other try yet, right?

Reply:
no, but akin to get_nvidia_path. We might improve it in the future